tests/tensor_search/test_on_start_script.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+from tests.marqo_test import MarqoTestCase
+from unittest import mock
+from marqo.tensor_search import enums, configs
+from marqo.tensor_search import on_start_script
+from marqo.s2_inference import s2_inference
+from marqo import errors
+
+
+class TestOnStartScript(MarqoTestCase):
+
+    def test_preload_models(self):
+        environ_expected_models = [
+            ({enums.EnvVars.MARQO_MODELS_TO_PRELOAD: []}, []),
+            ({enums.EnvVars.MARQO_MODELS_TO_PRELOAD: ""}, []),
+            (dict(), configs.default_env_vars()[enums.EnvVars.MARQO_MODELS_TO_PRELOAD]),
+            ({enums.EnvVars.MARQO_MODELS_TO_PRELOAD: ["onnx/all_datasets_v4_mpnet-base"]},
+             ["onnx/all_datasets_v4_mpnet-base"]),
+            ({enums.EnvVars.MARQO_MODELS_TO_PRELOAD: json.dumps(["onnx/all_datasets_v4_mpnet-base"])},
+             ["onnx/all_datasets_v4_mpnet-base"]),
+            ({enums.EnvVars.MARQO_MODELS_TO_PRELOAD: ["onnx/all_datasets_v4_mpnet-base", "hf/all_datasets_v3_mpnet-base"]},
```

Comment:
lets use the non-mpnet onnx models as the mpnet are not yet fully supported. the list here has https://marqo.pages.dev/Models-Reference/dense_retrieval/#onnx the available ones.

Reply:
and maybe add a ViT-B/32 instead